lib/matplotlib/dviread.py

```diff
@@ -294,9 +296,24 @@ def _read(self):
         Read one page from the file. Return True if successful,
         False if there were no more pages.
         """
+        # Pages appear to start with the sequence
+        #   bop (begin of page)
+        #   xxx comment
+        #   down
+        #   push
+        #     down, down
+        #     push
+        #       down (possibly multiple)
+        #       push  <=  here, v is the baseline position.
+        #         etc.
+        # (dviasm is useful to explore this structure.)
+        self._baseline_v = None
```

Comment:
do we want to put this in `__init__` as well?  Are we sure that `_output` will never be called without `_read` being called first?

Reply:
we'd already have errored out earlier anyways, because self.text wouldn't have been defined either.